src/libs/OptionsListUtils.ts

```diff
@@ -1725,7 +1724,7 @@ function getUserToInviteOption({
         isInSelectedOption ||
         (!isValidEmail && !isValidPhoneNumber) ||
         isInOptionToExclude ||
-        (isChronosEmail && !Permissions.canUseChronos(betas)) ||
+        isChronosEmail ||
```

Comment:
```suggestion
```
This condition evaluates to `false` after removing the chronos beta (`isChronosEmail && !true`). Thus it should be omitted.

Also please update the `getUserToInviteOption` comment (remove reference to chronos)